File: src/main/java/com/rs/java/game/area/banks/ArdougneSouthBank.java
package com.rs.java.game.area.banks;

import com.rs.java.game.WorldTile;
import com.rs.java.game.area.Area;
import com.rs.java.game.area.shapes.Polygon;

public final class ArdougneSouthBank extends Area {

	public ArdougneSouthBank() {
		super(
				new Polygon(new WorldTile[] {
						new WorldTile(2649, 3280, 0),
						new WorldTile(2649, 3287, 0),
						new WorldTile(2658, 3287, 0),
						new WorldTile(2658, 3280, 0),
						new WorldTile(2649, 3280, 0) // closed polygon
				})
		);
	}

	@Override
	public Area update() {
		return this;
	}

	@Override
	public String name() {
		return "Ardougne South Bank";
	}

	@Override
	public boolean member() {
		return false;
	}

	@Override
	public Environment environment() {
		return Environment.SAFEZONE;
	}
}

File: src/main/java/com/rs/kotlin/game/world/area/areas/banks/ArdougneSouthBank.kt
package com.rs.kotlin.game.world.area.areas.banks

import com.rs.java.game.WorldTile
import com.rs.kotlin.game.world.area.Area
import com.rs.kotlin.game.world.area.shape.Polygon

class ArdougneSouthBank : Area(
    Polygon(
        arrayOf(
            WorldTile(2649, 3280, 0),
            WorldTile(2649, 3287, 0),
            WorldTile(2658, 3287, 0),
            WorldTile(2658, 3280, 0),
            WorldTile(2649, 3280, 0) // closed polygon
        )
    )
) {
    override fun update(): Area {
        return this
    }

    override fun name(): String {
        return "Ardougne South Bank"
    }

    override fun member(): Boolean {
        return false
    }

    override fun environment(): Environment {
        return Environment.SAFEZONE
    }
}
